packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/OutputEditor.js

```diff
@@ -11,8 +11,14 @@ export default class OutputEditor extends Component {
 
     this.translate = context.injector ? context.injector.get('translate') : noopTranslate;
 
+    this.reformatName = (name) => {
+      return name.replace(/\W+(.)/g, (match, firstChar) => {
+        return firstChar.toUpperCase();
+      });
+    };
```

Comment:
Citing the requirement from the issue

> Add a validation to the output name field and check for whitespaces. Output name must not contain whitespaces.

I think we won't like to overrule the user by simply updating the set name. Instead, validation would be appreciated.

Reply:
Can you describe the validation? Like after input, warn the user that the name must not contain space or something?
Also how do I pass the codecov test specifically? I did pull the latest code before making any change.

Reply:
> Can you describe the validation? Like after input, warn the user that the name must not contain space or something?

Yes, some basic input validation would do the trick.

> Also how do I pass the codecov test specifically?

Codecov tests whether your added code is covered by test cases. So you'll have to add some to verify your feature is working properly.